Description: Modify output of TrapezoidalFactory tests.
 There is a bug in GCC 4.8 on *-i386 architectures, several tests
 hang.  In order to avoid these problems, algocobyla.c is compiled
 without optimization; but those tests must be modified.
 This patch is applied only on *-i386 architectures.
Author: Denis Barbier
Origin: Debian
Forwarded: no
Last-Update: 2013-07-02

--- openturns-1.1.orig/lib/test/t_TrapezoidalFactory_std.expout
+++ openturns-1.1/lib/test/t_TrapezoidalFactory_std.expout
@@ -1,8 +1,8 @@
 Distribution          =class=Trapezoidal name=Trapezoidal dimension=1 a=1 b=2.3 c=4.5 d=5 h=0.322581
-Estimated distribution=class=Trapezoidal name=Trapezoidal dimension=1 a=1.006 b=2.275 c=4.545 d=4.99 h=0.3198
+Estimated distribution=class=Trapezoidal name=Trapezoidal dimension=1 a=1.005 b=2.279 c=4.545 d=4.99 h=0.32
 Default distribution=class=Trapezoidal name=Trapezoidal dimension=1 a=-2 b=-1 c=1 d=2 h=0.333333
 Distribution from parameters=class=Trapezoidal name=Trapezoidal dimension=1 a=1 b=2.3 c=4.5 d=5 h=0.322581
 Trapezoidal          =class=Trapezoidal name=Trapezoidal dimension=1 a=1 b=2.3 c=4.5 d=5 h=0.322581
-Estimated trapezoidal=class=Trapezoidal name=Trapezoidal dimension=1 a=1.006 b=2.275 c=4.545 d=4.99 h=0.3198
+Estimated trapezoidal=class=Trapezoidal name=Trapezoidal dimension=1 a=1.005 b=2.279 c=4.545 d=4.99 h=0.32
 Default trapezoidal=class=Trapezoidal name=Trapezoidal dimension=1 a=-2 b=-1 c=1 d=2 h=0.333333
 Trapezoidal from parameters=class=Trapezoidal name=Trapezoidal dimension=1 a=1 b=2.3 c=4.5 d=5 h=0.322581
--- openturns-1.1.orig/python/test/t_TrapezoidalFactory_std.expout
+++ openturns-1.1/python/test/t_TrapezoidalFactory_std.expout
@@ -1,8 +1,8 @@
 distribution= class=Trapezoidal name=Trapezoidal dimension=1 a=1 b=2.3 c=4.5 d=5 h=0.322581
-Estimated distribution= class=Trapezoidal name=Trapezoidal dimension=1 a=1.006 b=2.275 c=4.545 d=4.99 h=0.3198
+Estimated distribution= class=Trapezoidal name=Trapezoidal dimension=1 a=1.005 b=2.279 c=4.545 d=4.99 h=0.32
 Default distribution= Trapezoidal(a = -2, b = -1, c = 1, d = 2)
 Distribution from parameters= Trapezoidal(a = 1, b = 2.3, c = 4.5, d = 5)
 Trapezoidal          = Trapezoidal(a = 1, b = 2.3, c = 4.5, d = 5)
-Estimated trapezoidal= Trapezoidal(a = 1.006, b = 2.275, c = 4.545, d = 4.99)
+Estimated trapezoidal= Trapezoidal(a = 1.005, b = 2.279, c = 4.545, d = 4.99)
 Default trapezoidal= Trapezoidal(a = -2, b = -1, c = 1, d = 2)
 Trapezoidal from parameters= Trapezoidal(a = 1, b = 2.3, c = 4.5, d = 5)

